test/jdk/java/awt/print/PrinterJob/PrintTranslatedFont.java

```diff
@@ -21,237 +21,127 @@
  * questions.
  */
 
-/**
+import java.awt.Dimension;
+import java.awt.Font;
+import java.awt.FontMetrics;
+import java.awt.Frame;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.Panel;
+import java.awt.geom.AffineTransform;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
+
+import javax.swing.JButton;
+import javax.swing.JOptionPane;
+
+/*
  * @test
  * @bug 6359734
  * @key printer
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
  * @summary Test that fonts with a translation print where they should.
  * @run main/manual PrintTranslatedFont
  */
+public class PrintTranslatedFont extends Frame {
+    private static final String INSTRUCTIONS =
+            "This test should print a page which contains the same\n" +
+            "content as the test window on the screen, in particular the lines\n" +
+            "should be immediately under the text\n" +
+            "You should also monitor the command line to see if any exceptions\n" +
+            "were thrown\n" +
```

Comment:
```suggestion
```
The tester cannot see the console or any messages printed onto the console by the test.